app/controllers/events_controller.rb

```diff
@@ -32,6 +32,7 @@ def create
 
   # PATCH/PUT /events/1
   def update
+    print event_params
```

Comment:
Shouldn't this be excluded, since it doesn't really serve any purpose except debugging?